dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -846,6 +846,20 @@ def wait_for_jquery
   }
 end
 
+Given(/^I sign out and sign in as "([^"]*)"$/) do |name|
+  steps %Q{
```

Comment:
could be useful to use `individual_steps` instead of `steps`, so that you can see which line it fails on